external/mbedtls/alt/pk_wrap_alt.c

```diff
@@ -964,6 +964,8 @@ int eckey_sign_wrap(void *ctx, mbedtls_md_type_t md_alg,
 		mbedtls_ecdsa_free(&ecdsa);
 		return MBEDTLS_ERR_ECP_INVALID_KEY;
 	}
+	key_idx = (((mbedtls_ecp_keypair *) ctx)->grp).key_index = (((mbedtls_ecp_keypair *) ctx)->key_index);
+
```

Comment:
`(((mbedtls_ecp_keypair *) ctx)->key_index)` is duplicated.

Reply:
are you pointing to, `ctx->grp.key_index` and `ctx->key_index`?

Reply:
updated